src/MLIAP/mliap_descriptor_so3.cpp

```diff
@@ -0,0 +1,298 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://lammps.sandia.gov/, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#include "mliap_descriptor_so3.h"
+#include "atom.h"
```

Comment:
there should be an empty line before `#include "atom.h" ` so that clang-format's sorting of include statements does not move the class header, which should be the very first include file.